scripts/mobile/android-wrap-asan.sh

```diff
@@ -0,0 +1,31 @@
+#!/system/bin/sh
```

Comment:
You can also have a `wrap.sh` that enables `malloc debug`:
* https://developer.android.com/ndk/guides/wrap-script#malloc_debug
* https://android.googlesource.com/platform/bionic/+/master/libc/malloc_debug/README.md